recipe/meta.yaml

```
@@ -1,17 +1,13 @@
-{% set version = "2019.9" %}
+{% set version = "2020.0" %}
 
 {% set year = version.split('.')[0]|int %}
 {% set update = version.split('.')[1]|int %}
 {% if version.split('.').__len__() == 2 %}
-{%  if update == 0 %}
-{%   set vtag = "%d" % year %}
-{%  else %}
-{%   set vtag = "%d_U%d" % (year, update) %}
-{%  endif %}
+{%  set vtag = "v%d.%d" % (year, update) %}
 {% else %}  # have to take into account for sake of the bot
 {%  set vtag = "malformed_" + version %}
 {% endif %}
-{% set vinterface = (year-2008)*1000 + update %}  # just guess, does not fit the previous versions before 2018
+{% set vinterface = (year-2009)*1000 + 100 + update %}  # just guess, does not fit the previous versions before 2020
```

hm.. my guess is that the next major update might want to remove `+100` from this formula.. if there is no way to find out how `+100` appears, I'd suggest just to write the "magic major" number explicitly. I.e.:
```suggestion
{% set vinterface = 11100 + update %}  # just guess for 2020 series of updates
```